models/classes/class.TestsService.php

```diff
@@ -165,7 +165,8 @@ public function cloneInstance(
         if (!is_null($clone)) {
             $noCloningProperties = [
                 self::PROPERTY_TEST_CONTENT,
-                OntologyRdf::RDF_TYPE
+                OntologyRdf::RDF_TYPE,
+                TaoOntology::PROPERTY_TRANSLATED_INTO_LANGUAGES
```

Comment:
```suggestion
                TaoOntology::PROPERTY_TRANSLATED_INTO_LANGUAGES,
```

Reply:
There may be other metadata we also should not clone, like the translation status, etc. The same for items. I think this is not causing issues with the current tickets, but maybe we should also add them?

Wdyt @shpran ?